pkg/test/step.go

```diff
@@ -36,7 +36,7 @@ func (t TestStepParameters) Get(k string) []Param {
 func (t TestStepParameters) GetOne(k string) Param {
 	v, ok := t[k]
 	if !ok || len(v) == 0 {
-		return Param("")
+		return Param{}
```

Comment:
We should probably return a pointer now

Reply:
No strong opinion, I can change it

Reply:
Sorry, I meant that the API should not return a copy but a value, since it's now a struct. It probably won't make a big difference from a performance perspective, but it's definitely not a pure literal anymore

Reply:
ok doing it